feature-flag/README.md

```diff
@@ -0,0 +1,126 @@
+# Feature flag
+
+## Configuration
+
+```ruby
+# config/initializers/flipper.rb
+require "flipper"
+require "flipper/adapters/memory"
+
+Feature.configure do |config|
+  config.flipper = Flipper.new(Flipper::Adapters::Memory.new)
+  config.dev_mode = ENV["FEATURE_DEV_MODE"].present?
+end
+```
+
+## Usage
+
+* In views and controllers: `feature?(:feature)`
+* In PORO: `Feature.enabled?(:feature, user)`
+
+
+## Implementation
+
+```ruby
+# Gemfile
+gem "flipper"
+
+# app/models/feature.rb
+class Feature
+  cattr_accessor :dev_mode
+  self.dev_mode = false
```

Comment:
I'd suggest to also add:
`alias :dev_mode? :dev_mode`

Then later in `enabled?`:
```ruby
return true if dev_mode?
```

Or, instead of alias, there may be a getter like this:
```ruby
def dev_mode
  !!dev_mode
end
```